test/Kestrel.Transport.FunctionalTests/ChromeTests.cs

```diff
@@ -52,6 +52,7 @@ private void InitializeArgs()
                 $"--no-sandbox " +
                 $"--disable-gpu " +
                 $"--allow-insecure-localhost " +
+                $"--ignore-certificate-errors --enable-features=NetworkService " +
```

Comment:
It seems that NetworkService is behind a feature flag because it's still experimental. If we see new issues, we should look at flipping this back off.